package.json

```diff
@@ -32,7 +32,10 @@
       "ts",
       "js"
     ],
-    "testRegex": "/src/__tests__/test\\..+\\.ts$",
+    "testRegex": [
+      "/src/__tests__/test\\..+\\.ts$",
+      "/src/__tests__/rtcp/test\\..+\\.ts$"
+    ],
```

Comment:
In the future we'll add more folders for jitter buffer, etc. Can we have a single string here (instead of an array) with the proper regex to cover all folders under `__tests__`?